Source/Core/DolphinQt/Main.cpp

```diff
@@ -132,8 +132,10 @@ int main(int argc, char* argv[])
   const optparse::Values& options = CommandLineParse::ParseArguments(parser.get(), argc, argv);
   const std::vector<std::string> args = parser->args();
 
+#if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
   QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
```

Comment:
hm, these 2 flags were deprecated in Qt6, but not clear to me if that implicitly makes some of the existing dolphin code behave badly. AA_UseHighDpiPixmaps was documented as:

> Make [QIcon::pixmap](https://doc.qt.io/qt-5/qicon.html#pixmap)() generate high-dpi pixmaps that can be larger than the requested size. Such pixmaps will have [devicePixelRatio()](https://doc.qt.io/qt-5/qpixmap.html#devicePixelRatio) set to a value higher than 1. After setting this attribute, application code that uses pixmap sizes in layout geometry calculations should typically divide by [devicePixelRatio()](https://doc.qt.io/qt-5/qpixmap.html#devicePixelRatio) to get device-independent layout geometry.

Reply:
should probably also check if the dpi setting in manifest should be changed/removed https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/DolphinQt/DolphinQt.manifest#L5 (Qt6 is takes care of it now)